dist/src/main/package/www/controllers/gateways.js

```diff
@@ -291,7 +291,11 @@ myControllerModule.controller('GatewaysControllerAddEdit', function ($scope, Typ
       $scope.gateway.pollFrequency='1';
       $scope.gateway.recordsLimit='10';
       $scope.gateway.trustHostType='';
-    }
+    }else if($scope.gateway.type === 'Philips Hue'){
+          $scope.gateway.url='http://<bridge ip address>';
```

Comment:
@Fraid We have to add some default IP or we can leave this as blank, MyController supports for multi-locale, which leads issue, when we put `bridge ip address` on url.